### PR TITLE
Explain what this means.

### PR DESCRIPTION
**Pull request by rosiel**
**JIRA Ticket**:  none, this is 8x. 

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?

Source field, JSON type alter - this needs documentation, it's currently "[an arcane backend process](https://islandora.github.io/documentation/islandora/rdf-mapping/)". I started by renaming the fields to actually say something to someone who isn't already eyeballs deep in how JSON-LD and RDF are created, and instructions to allow someone to make use of this field.

# What's new?

Explain what to expect from this field setting. Explain what is required for this field setting to work. 

# How should this be tested?

Does it make sense? 

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* Does this change the interface, add a new feature, or otherwise change behaviours that would require updating documentation? - Yes, a whole section on what this means/does will need to be added. 
* Does this change add any new dependencies? - No
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? - No
* Could this change impact execution of existing code? - Shouldn't.

# Interested parties

@Islandora/8-x-committers

### CODE REVIEW

**Comment by rosiel** [comment]
Screen Shot 2020-09-03 at 4 26 00 PM

File: src/Plugin/ContextReaction/JsonldTypeAlterReaction.php [hunk 1]
```diff
@@ -89,9 +89,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
     $config = $this->getConfiguration();
     $form['source_field'] = [
       '#type' => 'select',
-      '#title' => $this->t('Source Field'),
+      '#title' => $this->t('Field containing RDF type information'),
       '#options' => $options,
-      '#description' => $this->t("Select the field containing the type predicates."),
+      '#description' => $this->t("In JSON-LD representations, allow the rdf:type to vary
```
Comment (jordandukart) [inline src/Plugin/ContextReaction/JsonldTypeAlterReaction.php]: As someone who hasn't mucked around with JSON-LD much this reads like a great improvement to me. Only thing I could suggest is potentially pointing out to some external documentation (if it exists) to further clarify.

Comment (rosiel) [reply src/Plugin/ContextReaction/JsonldTypeAlterReaction.php]: When i've put this in, i'd be happy to write such external docs, and provide a link.  "arcane backend process" is the only place i saw it described in the existing documentation repo.

File: src/Plugin/ContextReaction/JsonldTypeAlterReaction.php [hunk 2]
```diff
@@ -89,9 +89,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
     $config = $this->getConfiguration();
     $form['source_field'] = [
       '#type' => 'select',
-      '#title' => $this->t('Source Field'),
+      '#title' => $this->t('Field containing RDF type information'),
```
Comment (jordandukart) [inline src/Plugin/ContextReaction/JsonldTypeAlterReaction.php:92]: Looks like the JSONLD suite of tests failed due to looking to target that title: https://travis-ci.org/github/Islandora/islandora/jobs/723897394#L1427-L1429.

Comment (rosiel) [reply src/Plugin/ContextReaction/JsonldTypeAlterReaction.php:92]: oops! Thanks @jordandukart!

**Comment by jordandukart** [comment]
Looks good @rosiel! You accidentally committed some `.swps`, when you remove those I'll approve/merge this.

**Comment by rosiel** [comment]
Aww dangit i thought I got rid of those. Thank you @jordandukart!

**Review by jordandukart — APPROVED** [review]

**Pull request merged**